src/test/features/claim/form/models/claimAmount.ts

```diff
@@ -95,29 +101,96 @@ describe('ClaimAmountBreakdown', () => {
   })
 
   describe('appendRow', () => {
-    let rows
 
-    beforeEach(() => {
-      rows = [
-        {
-          reason: '',
-          amount: undefined
-        }
-      ]
-    })
+    it('adds empty element to list of rows', () => {
+      const actual: ClaimAmountBreakdown = new ClaimAmountBreakdown()
 
-    it('should append one row when called once', () => {
-      let breakdown = new ClaimAmountBreakdown(rows)
-      breakdown.appendRow()
-      expect(breakdown.rows).to.have.lengthOf(2)
+      expect(actual.rows.length).to.be.eq(INIT_ROW_COUNT)
+
+      actual.appendRow()
+
+      expect(actual.rows.length).to.be.eq(INIT_ROW_COUNT + 1)
     })
 
     it('should append three row when called thrice', () => {
-      let breakdown = new ClaimAmountBreakdown(rows)
+      let breakdown = new ClaimAmountBreakdown([])
       breakdown.appendRow()
       breakdown.appendRow()
       breakdown.appendRow()
-      expect(breakdown.rows).to.have.lengthOf(4)
+      expect(breakdown.rows).to.have.lengthOf(3)
+    })
+
+    it('adds only up to 20 elements', () => {
+      const actual: ClaimAmountBreakdown = new ClaimAmountBreakdown()
+
+      expect(actual.rows.length).to.be.eq(INIT_ROW_COUNT)
+
+      for (let i = 0; i < 100; i++) {
```

Comment:
Could be `for (let i = 0; i < INIT_ROW_COUNT + 1; i++) {`, then no need to maintain when limit increased.

Reply:
ok